### DISCUSSION TITLE
How can I apply font(*.ttf) when convert py to exe with onefile?

### DISCUSSION THREAD

**Discussion by ufsaa2022** [brentvollebregt/auto-py-to-exe#375] [Q&A]
hello sir,

I'm creating an exe file via auto-py-to-exe.
However, the font was applied to the file created with 'One Directory', but the font was not applied to the file created with 'One File'.
I used a function like 'resource_path' and added *.ttf to Additional Files, but it didn't apply.
Is there any other way?

Best Regards,

**Comment by brentvollebregt** [comment]
Have you added the *.tff file in the additional files section? The packaging process can only auto-detect Python imports
